src/main/java/pl/javastart/voting/Voting.java

```diff
@@ -27,8 +24,31 @@ public static void main(String[] args) {
      * Metoda powinna pobrać głos dla każdego przekazanego głosującego i zapisać wyniki głosowania do VotingResult
      */
     VotingResult executeVoting(List<String> voters, Scanner scanner) {
-
-        return null; // to możesz (a nawet powinieneś/powinnaś) zmienić :)
+        VotingResult votingResult = new VotingResult();
+        for (String voter : voters) {
+            String decision;
+            do {
+                System.out.println("Jak głosuje " + voter + "? (z - za, p - przeciw, w - wstrzymanie się)");
+                String option = getOption(scanner);
+                decision = option;
+            } while (decision.equals(""));
+            Vote vote = new Vote(voter, decision);
+            votingResult.votes.add(vote);
```

Comment:
Tutaj zdecydowanie lepiej wyglądałaby metoda `votingResult.addVote(vote)` - wtedy ukrywasz wewnętrzną strukturę `VotingResult` i nie przywiązujesz się do tego, że musi to być lista